specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/CommonDefinitions.json

```diff
@@ -1185,6 +1185,204 @@
+          "description": "Timeout, in seconds, for a request to respond",
+          "format": "int32",
+          "type": "integer"
+        },
+        "connectionTimeoutInSeconds": {
+          "description": "Timeout, in seconds, for a request to initiate a connection",
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "HttpRetryPolicy": {
+      "description": "Policy that defines http request retry conditions",
+      "type": "object",
+      "properties": {
+        "maxRetries": {
+          "description": "Maximum number of times a request will retry",
+          "format": "int32",
+          "type": "integer"
+        },
+        "retryBackOff": {
+          "description": "Settings for retry backoff characteristics",
+          "type": "object",
+          "properties": {
+            "initialDelayInMilliseconds": {
+              "description": "Initial delay, in milliseconds, before retrying a request",
+              "format": "int64",
+              "type": "integer"
+            },
+            "maxIntervalInMilliseconds": {
+              "description": "Maximum interval, in milliseconds, between retries",
+              "format": "int64",
+              "type": "integer"
+            }
+          }
+        },
+        "matches": {
+          "description": "Conditions that must be met for a request to be retried",
+          "type": "object",
+          "properties": {
+            "headers": {
+              "description": "Headers that must be present for a request to be retried",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/HeaderMatch"
+              }
+            },
+            "httpStatusCodes": {
+              "description": "Additional http status codes that can trigger a retry",
+              "type": "array",
+              "items": {
+                "format": "int32",
+                "type": "integer"
+              }
+            },
+            "errors": {
+              "description": "Errors that can trigger a retry",
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    },
+    "TcpRetryPolicy": {
+      "description": "Policy that defines tcp request retry conditions",
+      "type": "object",
+      "properties": {
+        "maxConnectAttempts": {
+          "description": "Maximum number of attempts to connect to the tcp service",
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "circuitBreakerPolicy": {
+      "description": "Policy that defines circuit breaker conditions",
+      "type": "object",
+      "properties": {
+        "consecutiveErrors": {
+          "description": "Number of consecutive errors before the circuit breaker opens",
+          "format": "int32",
+          "type": "integer"
+        },
+        "intervalInSeconds": {
```

Comment:
The description already references return to service. Is there a way you'd like it called out more explicitly? I know we tend to hide envoy from end-users so I didn't want to reference anything envoy related directly.

This is also the only place where we would have a default, isn't it? Is it worth calling out or should we enforce that customers always set it?

Reply:
> The description already references return to service

It's not just return to service, it can also eject a replica from service. Maybe something like:

"The time interval between endpoint checks. This can result in opening the circuit breaker if the check fails as well as closing the circuit breaker if the check succeeds. Defaults to 10s."

Reply:
Gotcha, makes sense. Updated.